components/collection/activity/ownerInsightsTabs/HolderTab.vue

```diff
@@ -71,7 +71,12 @@ const toggleNFTDetails = (flipperId: string) => {
   }
 }
 
-const holders = computed(() => Object.entries(props.owners || {}))
+const holders = computed(() =>
+  Object.entries(props.owners || {}).sort(
+    // sort by nft count: highest first
+    (a, b) => b[1].nftCount - a[1].nftCount
+  )
+)
```

Comment:
I do not honestly know what is data flow here but would be far better to do this on backend